README.md

```diff
@@ -210,6 +210,26 @@ code can be viewed in Chrome 76+ with the command-line flag
 `--enable-blink-features=LayoutInstabilityAPI`, or in Chrome 73-75 with the
 command-line flag `--enable-blink-features=LayoutJankAPI`.
 
+### Limitations
+
+The presence of "layout instability" as defined by this metric correlates
+only imperfectly with the user experience of "jumpy" websites.
+
+It's possible for a website to seem jumpy, but score well on CLS.  For example,
+rebuilding the DOM with entirely new elements does not trigger a layout shift.
+
+Conversely, it's possible for a website to provide a smooth user experience, but
+score poorly on CLS.  For example, an image carousel that animates a layout
+property such as [`left`](https://developer.mozilla.org/en-US/docs/Web/CSS/left)
+will produce a layout shift on every frame of the animation.  (Carousel authors are
```

Comment:
No need to use parenthesis on a full sentence. Also, is there any other benefit of using transform instead, other than improving the CLS score? If there's not, then it could be worded as 'Carousel authors could use transform[...]'

Reply:
The parentheses are there because the guidance about using transform is tangential to the purpose of the paragraph, which is to advise the reader that CLS is unnaturally inflated by layout-inducing animations.

There is another benefit to using transform: it enables compositing.  I have updated the sentence to mention it.